wimaging/wimaging_script_winpe_setup.erb

```diff
@@ -32,12 +36,12 @@ C:\Windows\System32\bcdboot C:\Windows /l en-US
  
 echo Downloading the drivers 
 md c:\drivers
-%WGET% -P c:\drivers -r -np -nH --cut-dirs=3 -R index.html -q --level=0 <%= @host.os.medium_uri(@host).to_s %>/drivers/
+%WGET% -P c:\drivers -r -np -nH --cut-dirs=<%= cut_dirs %> -R index.html -q --level=0 <%= @host.os.medium_uri(@host).to_s %>/drivers/
```

Comment:
I needed to bump the number to --cut-dirs 5 so that drivers/extras were in the right place on the deployed host. While adjusting this i thought that maybe someone wants to have drivers/extra packages in a different subdirectory so i made this option configurable. Even better would be the possibility to define a completely custom path to those resources. What do you think?